src/pybind/mgr/dashboard/frontend/src/app/shared/components/card-row/card-row.component.html

```diff
@@ -169,3 +194,38 @@
     <ng-template ngPluralCase="other">{{ title }}s</ng-template>
   </span>
 </ng-template>
+
+<ng-template #dropdownTemplate>
+  <ng-container *ngFor="let data of dropdownData.total.category | keyvalue">
+    <li class="list-group-item">
+      <div class="d-flex pl-1 pb-2 pt-2">
+        <div class="ms-5 me-auto">
+          <span *ngIf="data.value.total"
+                [ngPlural]="data.value.total">
+              {{ data.value.total }}
+            <ng-template ngPluralCase="=0">{{ hwNames[data.key] }}</ng-template>
+            <ng-template ngPluralCase="=1">{{ hwNames[data.key] }}</ng-template>
+            <ng-template ngPluralCase="other">{{ hwNames[data.key] }}s</ng-template>
+          </span>
+        </div>
+        <span *ngIf="data.value.ok"
+              class="me-3">
+          <span>
```

Comment:
this looks like an unnecessary span. BTW, there was some styling issue here which I fixed in the patch I put here somewhere

Reply:
this is still an issue

Reply:
addressed